packages/statemanager/package.json

```diff
@@ -64,7 +64,6 @@
     "@ethereumjs/block": "^5.3.0",
     "@ethereumjs/genesis": "^0.2.3",
     "@types/debug": "^4.1.9",
-    "rustbn-wasm": "^0.4.0",
-    "verkle-cryptography-wasm": "^0.4.8"
+    "rustbn-wasm": "^0.4.0"
```

Comment:
This also seems to be not needed anymore?

Reply:
Yes, you are right, not sure how this came in here in the first place? Anyhow, have directly removed via GitHub online editor and committed.